### ISSUE TITLE
Strip emoji's from changelog heading

### ISSUE THREAD

**Issue by HonkingGoose** [Belco90/octoclairvoyant#66]
## Problem

Changelogs can use emojis like 🐛 🏠 📝 🚀.
The `octoclairvoyant` site does not show these emoji's in the heading, they look like this:

`:bug: Bug Fix`

Even if we did show emoji's they will clutter up the changelog view with unnecessary color/graphics.
Emoji's will also make sorting the changelogs way harder, as we need to account for emoji use in the headings.

## Solution

I propose we strip any `:emoji-name:` from the heading before we show it in the end-user view.

**Comment by Belco90** [comment]
Mmm interesting one! Definitely, nice to have. Are you interested in working on this yourself?

**Comment by Belco90** [comment]
By the way: the example you attached is not grouping the different levels of changes properly. It's receiving the changes headings in a non-expected structure, so I think that should be also improved to get the changelog grouped by change levels instead of by date (as it is now).

**Comment by HonkingGoose** [comment]
> Mmm interesting one! Definitely, nice to have. Are you interested in working on this yourself?

No, I don't know enough JavaScript to figure out how it should work. 🙈 
But thank you for offering! 😉
